src/config.ts

```diff
@@ -0,0 +1,12 @@
+export default {
+  algorithm: 'aes-256-ctr',
+  preloader: {
+    username: 'preloader',
+    address: 'AYeceuGa7tTsyG6jgq7X6qKdoXt9iJJKN6',
+    passphrase: 'window perfect fiber whip source stock tower upset crucial later letter social'
```

Comment:
Never use hardcoded values. always use an `.env` file that is under version control to avoid things like committing passwords and certificates.

Reply:
https://github.com/ArkEcosystem/poc-ark-messenger/pull/7